src/olympia/editors/tests/test_views.py

```diff
@@ -2999,6 +3001,9 @@ def test_auto_approval_summary(self):
             'Has too few consecutive human-approved updates.')
         assert (
             doc('.auto_approval li').eq(1).text() ==
+            'Has too many daily users.')
+        assert (
+            doc('.auto_approval li').eq(2).text() ==
             'Uses a custom CSP.')
```

Comment:
Test for it _not_ being displayed for non-auto-approvals?  (The issue this pull is purporting to fix.)